Description: Disable tests using unpublished internal test crate
 which in addition uses downloaded test data (tests/data/build.rs upstream).
Last-Update: 2024-12-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/test_emitter.rs
+++ b/tests/test_emitter.rs
@@ -1,2 +1,3 @@
 #![allow(clippy::type_complexity, clippy::uninlined_format_args)]
+#![allow(dead_code)]
 
@@ -46,2 +47,2 @@
 
-libyml_test_suite::test_emitter!();
+//libyml_test_suite::test_emitter!();
--- a/tests/test_parser.rs
+++ b/tests/test_parser.rs
@@ -4,2 +4,3 @@
 #![allow(clippy::type_complexity, clippy::uninlined_format_args)]
+#![allow(dead_code)]
 
@@ -483,2 +484,2 @@
 // Run the test suite for parser
-libyml_test_suite::test_parser!();
+//libyml_test_suite::test_parser!();
--- a/tests/test_parser_error.rs
+++ b/tests/test_parser_error.rs
@@ -4,2 +4,3 @@
 #![allow(clippy::type_complexity, clippy::uninlined_format_args)]
+#![allow(dead_code)]
 
@@ -73,2 +74,2 @@
 // Run the test suite for parser errors
-libyml_test_suite::test_parser_error!();
+//libyml_test_suite::test_parser_error!();
